sklearn/utils/estimator_checks.py

```diff
@@ -1952,6 +1959,8 @@ def check_class_weight_balanced_linear_classifier(name, Classifier):
         classifier.set_params(n_iter=1000)
     if hasattr(classifier, "max_iter"):
         classifier.set_params(max_iter=1000)
+    if hasattr(classifier, 'cv'):
+        classifier.set_params(cv=3)
```

Comment:
Do we need this even with the changes to `set_checking_parameters`?

Reply:
Yes this test is not including in `check_estimator` and does not call `set_checking_parameters`.